### PR TITLE
Feature/check package versions

### PR DESCRIPTION
**Pull request by pchorus**
Second try. Now I made sure that the action works correctly in GitHub Actions.

Fixes for repositories to fullfil requirements for poetry package versions follow shortly.

### CODE REVIEW

**Comment by pchorus** [comment]
Created ticket MD-7262 to track these changes.

**Review by catcombo — APPROVED** [review]

**Review by mm-matthias — APPROVED** [review]

File: ruff.toml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# https://docs.astral.sh/ruff/configuration/
```
Comment (marns93) [inline ruff.toml:1]: This file is managed by Pulumi and will be deployed with your changes in https://github.com/moneymeets/moneymeets-pulumi/pull/572

File: .idea/watcherTasks.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (marns93) [inline .idea/watcherTasks.xml:1]: This file is managed by Pulumi and will be deployed with your changes in https://github.com/moneymeets/moneymeets-pulumi/pull/572

**Review by marns93 — CHANGES_REQUESTED** [review]
Please remove the files which are managed by Pulumi.

**Review by marns93 — APPROVED** [review]

**Comment by pchorus** [comment]
There are even more repositories that needed updates to their `pyproject.toml`. I hope I changed them all accordingly.

**Pull request merged**